### DISCUSSION TITLE
Where are the social icons from? They look really clean and I'd like to add some others as well.

### DISCUSSION THREAD

**Discussion by rss2086** [arthelokyo/astrowind#63] [Q&A] [answered]
First off! The template is excellent and saved me a ton of time. I'm at the point where I want to add more social links such as youtube, LinkedIn, etc. I'd like to keep the theme of the original template, so I was wondering if you could point me to the icon library used for the icons in src/components/icons.

Thanks!!

**Comment by vasfvitor** [comment, marked as answer]
I'm not the developer of this template.

The icons is the footer per the file: https://github.com/onwidget/astrowind/blob/6a1401a771fcee58429e6850ade0a857668085ee/src/components/widgets/Footer.astro
lines 52-59
```
const social = [
	{ label: 'Twitter', icon: 'tabler:brand-twitter', href: '#' },
	{ label: 'Instagram', icon: 'tabler:brand-instagram', href: '#' },
	{ label: 'Facebook', icon: 'tabler:brand-facebook', href: '#' },
	{ label: 'RSS', icon: 'tabler:rss', href: getRelativeLink('/rss.xml') },
	{ label: 'Github', icon: 'tabler:brand-github', href: 'https://github.com/onwidget/astrowind' },
];
```
Are from [Tabler icons](https://tablericons.com/), as it is most others from the template. 
This template uses [Astro-Icon](https://github.com/natemoo-re/astro-icon#readme) which is powered by Iconify: https://icon-sets.iconify.design/tabler/ so it easy to just copy the reference like for example youtube: tabler:brand-youtube. And put it in the template using the `<Icon>` component.

What do you mean by `src/components/icons`?

**Comment by mhaidarhanif** [comment]
Indeed, the template is the most excellent out of all Astro templates! Saved me too a lot of time. Especially the default icon set component is using Astro Icon: https://astroicon.dev

The social icons are bundled and coming from [Iconify](https://iconify.design), and to search the actual set of icons and their names using [Icones](https://icones.js.org) is so far the best and fastest.

For example, to get this icon name, get from <https://icones.js.org/collection/tabler?s=rss>

```
<Icon name="tabler:rss" />
```
